docs/rules/extensions.md

```diff
@@ -109,6 +109,8 @@ import foo from './foo';
 import bar from './bar';
 
 import Component from './Component';
+
+import foo from '@/foo'
```

Comment:
this isn't correct - the tests currently ensure that with "always", this should indeed omit the extension

Reply:
@ljharb [This test](https://github.com/import-js/eslint-plugin-import/blob/b236748e39d97dae4be873e06b77c411e178b654/tests/src/rules/extensions.js#L507-L523) will throw.

Reply:
@ljharb [This test](https://github.com/import-js/eslint-plugin-import/blob/b236748e39d97dae4be873e06b77c411e178b654/tests/src/rules/extensions.js#L507-L523) will throw.

Reply:
and this one won't https://github.com/import-js/eslint-plugin-import/blob/b236748e39d97dae4be873e06b77c411e178b654/tests/src/rules/extensions.js#L342 - maybe due to `ignorePackages` though? If so, let's be sure to include this example under that section as well.

Reply:
@ljharb Seems like it has been added:

https://github.com/import-js/eslint-plugin-import/blob/b743a650e9d474fcd5a4252b01c522adcc11b65c/tests/src/rules/extensions.js#L352-L356

Reply:
@ljharb What else do I need to do?

Reply:
I meant to add something in the rule docs :-)

Reply:
I've added two more cases:

- https://github.com/import-js/eslint-plugin-import/pull/2140/commits/99f31587b18854e05959a1ede2cca1cc7e2772ce#diff-e936099f904c0062296b824ed9e27946c5da628c70d5476d55a132869ea67183R152
- https://github.com/import-js/eslint-plugin-import/pull/2140/commits/99f31587b18854e05959a1ede2cca1cc7e2772ce#diff-e936099f904c0062296b824ed9e27946c5da628c70d5476d55a132869ea67183R164

Reply:
@ljharb I've added two more cases:

- https://github.com/import-js/eslint-plugin-import/pull/2140/commits/99f31587b18854e05959a1ede2cca1cc7e2772ce#diff-e936099f904c0062296b824ed9e27946c5da628c70d5476d55a132869ea67183R152
- https://github.com/import-js/eslint-plugin-import/pull/2140/commits/99f31587b18854e05959a1ede2cca1cc7e2772ce#diff-e936099f904c0062296b824ed9e27946c5da628c70d5476d55a132869ea67183R164